cypress-tests/cypress/e2e/PaymentUtils/Novalnet.js

```diff
@@ -0,0 +1,193 @@
+const successfulNo3DSCardDetails = {
+    card_number: "4000000000001091",
+    card_exp_month: "12",
+    card_exp_year: "25",
+    card_holder_name: "Joseph Doe",
+    card_cvc: "123",
+  };
+
+const successfulThreeDSTestCardDetails = {
+  card_number: "4000000000001091",
+  card_exp_month: "12",
+  card_exp_year: "25",
+  card_holder_name: "Joseph Doe",
+  card_cvc: "123",
+  };
+  
+  
+  export const connectorDetails = {
+    card_pm: {
+      PaymentIntent: {
+        Request: {
+          currency: "EUR",
+          customer_acceptance: null,
+          setup_future_usage: "on_session",
+        },
+        Response: {
+          status: 200,
+          body: {
+            status: "requires_payment_method",
+          },
+        },
+      },
+      "3DSManualCapture": {
+        Request: {
+          payment_method: "card",
+          payment_method_data: {
+            card: successfulThreeDSTestCardDetails,
+          },
+          currency: "EUR",
+          customer_acceptance: null,
+          setup_future_usage: "on_session",
+        },
+        Response: {
+          status: 200,
+          trigger_skip: true,
+          body: {
+            status: "requires_capture",
+          },
+        },
+      },
+      "3DSAutoCapture": {
+        Request: {
+          payment_method: "card",
+          payment_method_data: {
+            card: successfulThreeDSTestCardDetails,
+          },
+          currency: "EUR",
+          customer_acceptance: null,
+          setup_future_usage: "on_session",
+        },
+        Response: {
+          status: 200,
+          trigger_skip: true,
```

Comment:
I can see that redirection flow in `redirectionHandler` is not added.  

CC: @likhinbopanna